client/icecc-create-env.in

```diff
@@ -434,7 +452,7 @@ fi
 
 # special case for weird multilib setups
 for dir in /lib /lib64 /usr/lib /usr/lib64; do
-    test -L $dir && cp -p $dir $tempdir$dir
+    test -L $dir && cp -d $dir $tempdir$dir
```

Comment:
-d is a gnu ism.  Try to stick to posix code so that we can be portible to other systems.  (FreeBSD doens't seem to support -d)

Reply:
Since this is trying to copy the symlink, `-P` seems to be the right option.